rosidl_typesupport_c/include/rosidl_typesupport_c/action_type_support_dispatch.h

```diff
@@ -0,0 +1,36 @@
+// Copyright 2016 Open Source Robotics Foundation, Inc.
```

Comment:
2018

Reply:
Actually I don't think this file is necessary like it is for messages and services. The action type support doesn't change with `rmw` implementation, so there's no need to have a dispatch.

Reply:
That's true, but don't we need it to get proper typesupport for the underlying services and messages?

Reply:
> That's true, but don't we need it to get proper typesupport for the underlying services and messages?

Yes, but that can use the dispatch on the generated messages and services. The generated action type support just needs to call  `get_service_type_support<>()` or `get_message_type_support<>()`, and in C `ROSIDL_GET_SRV_TYPE_SUPPORT()` or `ROSIDL_GET_MSG_TYPE_SUPPORT()`

Reply:
Right, I realized shortly after I commented.